web/profiles/custom/os2loop/modules/os2loop_upvote/src/Helper/FlagHelper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\os2loop_upvote\Helper;
+
+use Drupal\Core\Cache\Cache;
+
+/**
+ * Flaghelper for creating top comment and count flags.
+ */
+class FlagHelper {
+
+  /**
+   * Find the top comment.
+   *
+   * If there is a comment marked as "correct answer", this is it.
+   * If there is no "correct answer", the top comment will be the most upvoted.
+   */
+  public function preprocessFlag(array &$variables) {
+    $flag_service = \Drupal::service('flag.count');
```

Comment:
Use dependency injection to get this service (see https://github.com/os2loop/os2loop/blob/feature/LOOP-748-documents-and-document-collections-no-menu/web/profiles/custom/os2loop/modules/os2loop_documents/os2loop_documents.services.yml#L5-L13 and https://github.com/os2loop/os2loop/blob/feature/LOOP-748-documents-and-document-collections-no-menu/web/profiles/custom/os2loop/modules/os2loop_documents/src/Helper/FormHelper.php#L70-L80 for an example).